pkg/agent/ipassigner/ip_assigner_linux.go

```diff
@@ -60,11 +61,19 @@ func NewIPAssigner(nodeTransportInterface string, dummyDeviceName string) (*ipAs
 		assignedIPs:       sets.NewString(),
 	}
 	if ipv4 != nil {
-		arpResonder, err := responder.NewARPResponder(externalInterface)
+		// Create an ARP responder if the dummy device does not exist or
```

Comment:
Question - why we do not always use ARP responder, and no longer assign IPs to the dummy device?

Reply:
Egress needs Egress IPs to be configured to host because Egress IPs are used as tunnel endpoints.

Reply:
Ok. Could we add a comment here, as it is not easy to figure out the implication.